eng/Build-Native.cmd

```diff
@@ -298,10 +298,12 @@ if %__Build% EQU 1 (
 
 REM Copy the native SOS binaries to where these tools expect for testing
 
-set "__dotnet_sos=%__RootBinDir%\bin\dotnet-sos\%__BuildType%\netcoreapp2.1\publish\win-%__BuildArch%"
-set "__dotnet_dump=%__RootBinDir%\bin\dotnet-dump\%__BuildType%\netcoreapp2.1\publish\win-%__BuildArch%"
-xcopy /y /q /i %__BinDir% %__dotnet_sos%
-xcopy /y /q /i %__BinDir% %__dotnet_dump%
+set "__dotnet_sos=%__RootBinDir%\bin\dotnet-sos\%__BuildType%\netcoreapp2.1\win-%__BuildArch%"
```

Comment:
Why did you change this? dotnet-dump expects them to be under "publish\win-xxx".

Reply:
Was it because dotnet-dump didn't work under VS? VS executes the dll under netcoreapp2.1 but when dotnet-dump is executed standalone it is from netcoreapp2.1\publish so the sos binaries need to be relative to that.

Reply:
This explains the test failures,  Did you run them locally? on a clean build?